docs/reference/release-notes/highlights.asciidoc

```diff
@@ -18,7 +18,7 @@ Other versions:
 | {ref-bare}/7.15/release-highlights.html[7.15]
 | {ref-bare}/7.14/release-highlights.html[7.14]
 | {ref-bare}/7.13/release-highlights.html[7.13]
-| {ref-bare}/7.11/release-highlights.html[7.12]
+| {ref-bare}/7.12/release-highlights.html[7.12]
```

Comment:
This appears to be an error that I noticed when testing the generation process.

Reply:
Good catch!